src/locales/ca/en/index.json

```diff
@@ -0,0 +1,34 @@
+{
+  "alreadyHaveAnAccount": "Already have an account? <a href='/'>Sign in here!</a>",
+  "dontHaveAnAccount": "Don’t have an account? <a href='/register'>Sign up here!</a>",
+  "closeModal": "Close Modal",
+  "chooseDifferentUsername": "Please choose a different username.",
+  "ensurePasswordIsMoreThan": "Ensure password contains more than {minPasswordLength} characters.",
+  "ensurePasswordHasOne": "Ensure password contains 1 of each: capital letter, number and special character.",
+  "givenName": "First Name",
+  "loading": "Loading ...",
+  "loginButton": "Login In",
+  "loginHeader": "Login In",
+  "loginSuccess": "Login Success!",
+  "mail": "Email Address",
+  "NameCallback": "User Name",
+  "nextButton": "Next",
+  "PasswordCallback": "Password",
+  "pleaseCheckValue": "Please check this value",
+  "preferencesmarketing": "Send me special offers and services",
+  "preferencesupdates": "Send me news and updates",
+  "registerButton": "Register",
+  "registerHeader": "Registration",
+  "registerSuccess": "Registration success!",
+  "securityAnswer": "Security Answer",
+  "securityQuestions": "Security question(s)",
+  "securityQuestionsPrompt": "Provide security question(s) & answer(s) below",
+  "showPassword": "Show Password",
+  "sn": "Last Name",
+  "submitButton": "Submit",
+  "successMessage": "Success!",
+  "termsAndConditions": "Please accept our Terms and Conditions",
+  "useValidEmail": "Please use a valid email address.",
+  "ValidatedCreatePasswordCallback": "Password",
+  "ValidatedCreateUsernameCallback": "Username"
```

Comment:
I was a little confused about what was being defined here at first - I can see some of these keys seem to correspond to the type of the callback, and others seem to be matching to what I think is probably the prompt for a callback, set in AM. Others seem to be keys created within the code based on how you are processing the validation policy failures. I guess my concern is that as an app and the flows implemented on AM grow, this could become quite hard to manage given the mix of things that are coupled to AM prompts, things that are not and things that only exist within this app. What would you think about structuring the json files in such a way as they separate "coupled FR stuff" from "non coupled FR stuff" e.g. callback types, and "app strings"?

Reply:
This is what I brought up in the demo two Fridays ago. When it comes to callbacks, localization is very difficult. Callbacks rarely have something to "key off of" for localized content. So, I did my best and used what was available to me.

I guess I could place a bit more responsibility in the callback handler to generate a more "normalized" key for the content, avoiding the rather heterogenous keys like what you see here. But, I'm open to suggestions.